src/services/iotHubManagerService.js

```diff
@@ -46,4 +48,12 @@ export class IoTHubManagerService {
     return HttpClient.delete(`${ENDPOINT}devices/${id}`)
       .map(() => ({ deletedDeviceId: id }));
   }
+
+    /** Returns the account's device group filters */
+    static getDeviceProperties() {
+      return Observable.forkJoin(
+      HttpClient.get(`${ENDPOINT}deviceProperties`).map(toDevicePropertiesModel),
```

Comment:
rather than hardcoding, we should have methods to return these URLs, in a reusable fashion

1. `${ENDPOINT}deviceProperties`
2. `${Config.serviceUrls.deviceSimulation}deviceProperties`

Since it's a cross-cutting concern across the entire codebase, I'd suggest to discuss this with @spryor

Reply:
it's also odd that in the first case the app uses a constant (`ENDPOINT = Config.serviceUrls.iotHubManager`) but in the second it references the config directly (`${Config.serviceUrls.deviceSimulation}`)

Reply:
@dluc Why do you think these should be wrapped in functions? The services classes serve as a single source of truth for service calls so urls (other than the configurable endpoints) should never change or be referenced directly outside these files. What is the benefit of adding an extra layer of complexity with functions?

Reply:
within the classes, I would define either a property or a method for each collection. For example, IoT Hub manager provides devices, deviceproperties, queries and jobs, so I'd consider adding something like:

devicesEndpoint = ${ENDPOINT}devices
devicePropertiesEndpoint = ${ENDPOINT}deviceProperties
queriesEndpoint = ${ENDPOINT}deviceProperties
jobsEndpoint = ${ENDPOINT}jobs

A separate observation, in the backend we code service classes by collection, while in the UI (if understand correctly) there is a service class per microservice. Have you considered having a "devices" class, a "deviceProperties" class, a "jobs" class etc?  In that case, each class would deal with a single endpoint.

(btw, these are not blocking considerations, just an observation about the project structure)